### ISSUE TITLE
nowjs 0.8.0 causes 404's to hang in express applications

### ISSUE THREAD

**Issue by rohansingh** [Flotype/now#172]
1. Create an express application:
   
   ```
   $ npm install -g express
   $ express /tmp/foo && cd /tmp/foo
   ```
2. Edit **package.json** to reference `"now": "0.8.0"`.
3. Install dependencies:
   
   ```
   $ npm install -d
   ```
4. Append this to **app.js**:
   
   ```
   var everyone = require('now').initialize(app);
   ```
5. Run the application:
   
   ```
   node app.js
   ```
6. Browse to a URL such as http://localhost:3000/should-get-404

Instead of getting back the expected 404, the request will hang. If you remove the line from step 4 or  use nowjs 0.7.6, the issue will disappear.

**Comment by ericz** [comment]
Interesting... looking into it.

**Comment by ericz** [comment]
This was a very critical bug. Fixed in 3691280cc06fc8ec2cbd7b177553717dff27b44a and version bumped to v0.8.1. `npm update now` to obtain working new version.
